### PR TITLE
You can now inject recording rules into our helm chart

### PR DESCRIPTION
**Pull request by duallain**
This is useful for boiling down very complex queries into easier to read simple queries
and was needed now for ctms - need to be able to compare a few numbers and it's easier with this method

### CODE REVIEW

**Review by floatingatoll — APPROVED** [review]

**Comment by duallain** [comment]
@cmharlow I'm giving your github action work a try on this pr - travis failed for what I think are unrelated reasons, so seemed better to just try this instead.

**Comment by cmharlow** [comment]
@duallain travis has been failing for a while, i believe a travis cred got removed. got my PR up to try to fix it.

**Comment by duallain** [comment]
It took me a minute to get here, but this should now pass CI and also do what I'd like with prom recording rules. @floatingatoll I invalidated your review, if you'd like to take a second look, that'd be appreciated.

File: charts/prometheus-customizations/templates/prometheus_config.yml [hunk 1]
```diff
@@ -45,6 +45,15 @@ spec:
...
         - /etc/config/alerts
+        {{- if .Values.prom.recording_rules }}
+        groups:
+        - name: recording_rules
+          rules:
+          {{- range $k, $v := .Values.prom.recording_rules }}
+          - record: {{ $k }}
+            expr: {{ $v }}
+          {{- end }}
+g        {{- end }}
```
Comment (floatingatoll) [inline charts/prometheus-customizations/templates/prometheus_config.yml:56]: I believe the 'g' here is a typo?

**Review by floatingatoll — APPROVED** [review]
This seems reasonable, with a syntax-error typo caveat below.

**Comment by floatingatoll** [comment]
(There's also a less-relevant typo in the commit message, I think.)

**Pull request merged**